test/unit/poller_test.rb

```diff
@@ -2,11 +2,9 @@
 
 describe 'Cron Poller' do
   before do
-    REDIS.with { |c| c.respond_to?(:redis) ? c.redis.flushdb : c.flushdb }
```

Comment:
Not sure what this `REDIS` thing is about. Sidekiq 7 doesn't support `Sidekiq.redis = ...`, seems `redis=` method exists on config, seemed most straightforward to gut this constant